File: 0x03_Array/cpp/10808.cpp
#include <iostream>

using namespace std;

int main()
{
    char word[101]; // 단어의 길이는 100을 넘지 않으며 모두 소문자.
    cin >> word;

    int counts[26] = {0};
    for (int i=0; word[i] != '\0'; i++)
    {
        // 아스키코드에서 'a'는 97, 'z'는 122 이므로 이를 counts의 인덱스로 사용한다.
        int idx = int(word[i]) - 97;
        counts[idx] += 1;
    }

    for (int i=0; i < 26; i++)
    {
        cout << counts[i] << " ";
    }

    return 0;
}

File: python/0x03_Array/10808.py
import sys
input = sys.stdin.readline

def solution():
    check = [0] * 26
    word = input().strip()
    for w in word:
        idx = ord(w) - 97 ## character의 아스키코드를 활용한다.
        check[idx] += 1

    result = ' '.join(map(str, check))
    print(result)

if __name__ == "__main__":
    solution()